packages/contracts/npm/artifacts.ts

```diff
@@ -45,10 +65,16 @@ export default {
   PluginRepoFactory,
   PluginRepoRegistry,
 
-  TokenVoting,
-  AddresslistVoting,
+  PluginSetupProcessor,
+
   Admin,
+  AdminSetup,
+  AddresslistVoting,
+  AddresslistVotingSetup,
+  TokenVoting,
+  TokenVotingSetup,
   Multisig,
+  MultisigSetup,
 
   MerkleMinter,
   MerkleDistributor,
```

Comment:
Needed?

Reply:
same here, they are not directly deployed by us, but since it can be deployed and used
let's keep them